site/content/docs/manifest/rd-web-service.en.md

```diff
@@ -137,6 +139,22 @@ Amount of memory in MiB reserved for each instance of your service. See the [AWS
 
 <div class="separator"></div>
 
+<a id="network" href="#network" class="field">`network`</a> <span class="type">Map</span>      
+The `network` section contains parameters for connecting the service to AWS resources in the environment's VPC.  
+By connecting the service to a VPC, you can use [service discovery](../developing/service-discovery.en.md) to communicate with other services
+in your environment, or connect to a database in your VPC with [`storage init`](../commands/storage-init.en.md).
```

Comment:
Previously it wasn't possible for Aurora only. Do you think it's worthwhile to emphasize on how it's possible now so that people that wanted this are aware?

Reply:
I was hoping that "connect to a database in your VPC" would be clear but maybe not, how about if I update like this?
> or connect to a database in your VPC such as Amazon Aurora with `storage init`